config/locales/en/shared.yml

```diff
@@ -564,11 +566,22 @@ en:
                 revealing_check_box_offline_savings_accounts: Include accounts with a negative balance or no money in them.
               offline_current_accounts: Enter the total in all current accounts
               offline_savings_accounts: Enter the total in all savings accounts
+            partners:
+              bank_accounts:
+                check_box_partner_offline_current_accounts: Current account
+                check_box_partner_offline_savings_accounts: Savings account
+                hint:
+                  revealing_check_box_partner_offline_current_accounts: Include accounts with a negative balance or no money in them.
+                  revealing_check_box_partner_offline_savings_accounts: Include accounts with a negative balance or no money in them.
+                partner_offline_current_accounts: Enter the total in all current accounts
+                partner_offline_savings_accounts: Enter the total in all savings accounts
             savings_and_investments:
```

Comment:
I find it confusing that this part is within `savings_and_investments` - I think I have actually removed all of this in my current PR. So there may be some merge conflicts for one/both of us at some point 😬 not sure where is best for it to go though!

Reply:
I think I had to add these for the CYA/review pages, but I can't remember now!